sdk/cosmos/azure_data_cosmos/src/models/mod.rs

```diff
@@ -54,23 +57,20 @@ impl<T: DeserializeOwned> azure_core::Model for QueryResults<T> {
 }
 
 /// A page of results from [`CosmosClient::query_databases`](crate::clients::CosmosClient::query_databases())
-#[non_exhaustive]
```

Comment:
Per ongoing internal discussions, we're removing `#[non_exhaustive]` from any types used as **request** models, since `#[non_exhaustive]` types _cannot be directly constructed outside the defining crate_ (which means we'd have to generate a whole bunch of constructors, setters, and builders). However, I removed them from all types here. I'm going back through and restoring them on the Response-only models in the next iteration. I'll wait to resolve this comment until that's done.